subxt/src/backend/unstable/rpc_methods.rs

```diff
@@ -745,11 +769,16 @@ pub enum TransactionStatus<Hash> {
 
 /// Details of a block that a transaction is seen in.
 #[derive(Debug, Clone, PartialEq, Eq, Deserialize)]
+#[cfg_attr(test, derive(Serialize))]
 pub struct TransactionBlockDetails<Hash> {
     /// The block hash.
     pub hash: Hash,
     /// The index of the transaction in the block.
-    #[serde(with = "unsigned_number_as_string")]
+    #[serde(deserialize_with = "unsigned_number_as_string::deserialize")]
```

Comment:
ok, deserialize is only implemented for this type and `with` is a shortcut for both serialize and deserialize impl